src/components/SystemIntakeReview/index.tsx

```diff
@@ -328,6 +370,8 @@ export const SystemIntakeReview = ({
             />
           </div>
         </ReviewRow>
+        {/* Format and conditionally render software acquisition information */}
+        {formatSoftwareAcquisition()}
```

Comment:
can this be a component instead?

```suggestion
        <SoftwareAcquisition />
```